Algorithms/Sorters/Comparison/CombSorter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Algorithms.Sorters.Comparison
+{
+    /// <summary>
+    /// Comb sort is a relatively simple sorting algorithm that improves on bubble sort.
+    /// </summary>
+    /// <typeparam name="T">Type of array element.</typeparam>
+    public class CombSorter<T> : IComparisonSorter<T>
+    {
+        /// <summary>
+        /// Sorts array using specified comparer,
+        /// internal, in-place, unstable,
+        /// time complexity: O(n^2),
+        /// space complexity: O(1),
+        /// where n - array length.
+        /// </summary>
+        /// <param name="array">Array to sort.</param>
+        /// <param name="comparer">Compares elements.</param>
+        public void Sort(T[] array, IComparer<T> comparer)
+        {
+            var gap = array.Length;
+            const double shrinkFactor = 1.3; // Suggested as an ideal shrink factor by the authors
```

Comment:
Could you move this into a property that is initialized in a constructor and create 2 constructors: one without parameters and it calls another passing in 1.3 as a parameter for shrink factor. Just in case someone wants to tweak the shrink factor for their particular situation.

Reply:
Good point, moved it to the constructor.